app/src/main/res/values/strings.xml

```diff
@@ -1,3 +1,8 @@
 <resources>
     <string name="app_name">alpine</string>
+    <string name="main_start">Start scanning MRZ</string>
+    <string name="rationale_permission_camera">To take the photo of the passport information page, you need to allow access to the camera.</string>
+    <string name="ok">OK</string>
+    <string name="no">No, thanks</string>
```

Comment:
Maybe rename this to `no_thanks` to better represent what the string is, what do you think? 🤔